homeassistant/components/sensor/const.py

```diff
@@ -597,6 +607,7 @@ class SensorStateClass(StrEnum):
 DEVICE_CLASS_STATE_CLASSES: dict[SensorDeviceClass, set[SensorStateClass]] = {
     SensorDeviceClass.APPARENT_POWER: {SensorStateClass.MEASUREMENT},
     SensorDeviceClass.AQI: {SensorStateClass.MEASUREMENT},
+    SensorDeviceClass.AREA: {SensorStateClass.MEASUREMENT},
```

Comment:
I think we should allow the integrating state classes here, `SensorStateClass.TOTAL` and `SensorStateClass.TOTAL_INCREASING` instead of, or possibly in addition to, `SensorStateClass.MEASUREMENT`.

See the developer documentation for details: https://developers.home-assistant.io/docs/core/entity/sensor#available-state-classes

For area mowed or area vacuumed, the total state classes are the better fit.

Reply:
Looks to me like we need all three.

Reply:
Done